guardian/typerighter
# Add rule manager subproject

## What does this change?

This PR adds a Rule Manager sub project to the Typerighter repo. It provides us with a starting point for developing the Rule Manager functionality. This PR includes the Play setup but doesn't go as far as to configure the deploy which will be covered in a separate PR. 

## How to test

From the perspective of the existing checker functionality, this should be a no-op. In terms of the Rule Manager it should be possible to run locally on [typerighter.local.dev-gutools.co.uk](https://typerighter.local.dev-gutools.co.uk/), this will require nginx to be setup again to account for the change of ports.

## How can we measure success?

We are able to proceed with the rule management functionality.
